app/components/decision/decision-result-pill.js

```diff
@@ -13,6 +13,20 @@ export default class DecisionResultPill extends Component {
     // } else if (selectedUri === codes.INGETROKKEN) {
     //  return 'danger';
     // }
-    return 'default';
+    return 'white';
+  }
+
+  get icon() {
+    const codes = CONSTANTS.DECISION_RESULT_CODE_URIS;
+    const selectedUri = this.args.decisionResultCode.get('uri');
+    if (selectedUri === codes.GOEDGEKEURD) {
+      return 'check';
+    } else if (selectedUri === codes.UITGESTELD) {
+      return 'clock';
+    }
+    // } else if (selectedUri === codes.INGETROKKEN) {
+    //  return 'danger';
+    // }
```

Comment:
copy of the skin getter, do we need this comment?
`danger` does not exist as an icon.